USAGE.md

```diff
@@ -365,6 +371,40 @@ Get-GitHubUser
 
 ----------
 
+### Repositories
+
+#### Get repository vulnerability alert status
+
+```powershell
+Get-GitHubRepositoryVulnerabilityAlert -OwnerName Microsoft -RepositoryName PowerShellForGitHub
```

Comment:
Can you update `OwnerName` to be lowercase `microsoft` here and elsewhere?

```suggestion
Get-GitHubRepositoryVulnerabilityAlert -OwnerName microsoft -RepositoryName PowerShellForGitHub
```

Reply:
Done